docs/index.rst

```diff
@@ -1,14 +1,125 @@
+
+If you're interested in contributing, check out the :doc:`Contributor Guide <sections/contributor_guide/index>`.
+
+For users who want to start using the toolbox and framework, take a peek at our :doc:`User Guide <sections/user_guide/index>`.
+
+.. contents:: On this page
+   :depth: 2
+   :local:
+
+
+
+The Tools
+---------
+
+All our tools are available by command line interface (CLI), with select operations supported in the Python API. The API will also give you access to the underlying object creation to do with those what you please. We hope these options will let you integrate the package into your pre-existing bash and Python scripts, and give you some handy tools to use in your day-to-day work with running NWP systems.
+
+Configuration Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+| **CLI**: ``uw config -h``
+| **API**: ``import uwtools.api.config``
+
+The uw config tool suite helps you compare, transform, modify, and even validate your configuration files. The package supports YAML, shell, Fortran namelist, and INI file formats. Each of these formats allows you to specify values as Jinja2 templates to enable referencing other values for direct substitution or derived information (math, path building, string manipulation, etc.).
+
+Compare Mode
+............
+
+When the Linux diff tool just doesn't work for comparing unordered namelists with mixed-case keys, this is your go-to! It also works on the other file types, but the Fortran namelists are the *real* catalyst behind this gem!
+
+| See CLI examples here: :any:`Examples<compare_configs_cli_examples>`
+| This mode is not supported in the API.
+
+
+Realize Mode
+............
+
+This mode does the work of rendering values created by Jinja2 templates, and lets you override settings in one file with those from others, not necessarily with the same file type. With ``uwtools``, you can even reference the content of other files to build up a configuration file from its pieces.
+
+| See CLI examples here: :any:`Examples<realize_configs_cli_examples>`
+
+Translate Mode
+..............
+This is a more limited tool that helps transform legacy configuration files templated with the atparse tool (common at NOAA) into a Jinja2 template for use with the uw config realize and uw template render tools.
+
+| See CLI examples here: :any:`Examples<translate_configs_cli_examples>`
+| This mode is not supported in the API.
+
+Validate Mode
+.............
+In this mode, you can provide a JSON Schema file alongside your configuration file to check to ensure that it meets the requirements set by the schema. We’ve enabled robust logging to make it easier to repair your config files when problems arise.
```

Comment:
An external link from "JSON Schema" to https://json-schema.org/ might be nice.

Maybe `to check to ensure that it` ("check" and "ensure" sound somewhat synonymous to me) -> `to validate that it`?

Maybe `configuration file` -> just `configuration`, as there may not be a file in play. In general, just `configs` or `configurations` are probably correct without adding `file`. (There are cases, like with the schema here, where a file really is required, but that detail may not be necessary.) I think it's worth staying general because, in addition to the CLI stdin/stdout cases, the API often operates on non-file objects like `dict`s, and I think `configs` covers files, streams, `dict`s, `Config` objects, in-memory strings, whatever we might be operating on.

Same note here about the API: This operation _is_ supported in the API. I think everything is, but please let me know if I'm missing something. You can verify via the auto-generated API docs.